plugins/aws/fix_plugin_aws/resource/ec2.py

```diff
@@ -392,7 +393,7 @@ class AwsEc2InstanceType(AwsResource, BaseInstanceType):
     _kind_service: ClassVar[Optional[str]] = service_name
     _metadata: ClassVar[Dict[str, Any]] = {"icon": "type", "group": "compute"}
     _aws_metadata: ClassVar[Dict[str, Any]] = {"arn_tpl": "arn:{partition}:ec2:{region}:{account}:instance/{id}"}  # fmt: skip
-    api_spec: ClassVar[AwsApiSpec] = AwsApiSpec(service_name, "describe-instance-types", "InstanceTypes")
+    # api_spec defined in `collect_resource_types` method and collected by AwsEc2Instance
```

Comment:
Can we leave it in here? 
Other option: you need to define this spec in `called_collect_apis`.